sql/core/src/main/scala/org/apache/spark/sql/catalyst/util/V2ExpressionBuilder.scala

```diff
@@ -267,6 +267,7 @@ class V2ExpressionBuilder(e: Expression, isPredicate: Boolean = false) extends L
     case _: DateAdd => generateExpressionWithName("DATE_ADD", expr, isPredicate)
     case _: DateDiff => generateExpressionWithName("DATE_DIFF", expr, isPredicate)
     case _: TruncDate => generateExpressionWithName("TRUNC", expr, isPredicate)
+    case _: TruncTimestamp => generateExpressionWithName("DATE_TRUNC", expr, isPredicate)
```

Comment:
is `DATE_TRUNC` a standard SQL function or widely supported in the industry?

Reply:
Not a standard function but it's supported by several databases - Postgres, Redshift, Snowflake, BQ